### ISSUE TITLE
Towing doesn't work

### ISSUE THREAD

**Issue by Falconinglasses** [CleverRaven/Cataclysm-DDA#76499]
### Describe the bug

As of 14 september version towing doesn't work at all, no matter what you attach to what, you end up with cable overextending and breaking loose.

### Attach save file

N/A

### Steps to reproduce

Attach anything to anything and try driving.

### Expected behavior

Expected 7 ton humvee with v8 to tow away wreck of superbike weighting 233 kg. 
Ended up ******* for an hour, and then used sledgehammer to break it. 
Then using debug tried to tow anything with anything, nothing worked.

### Screenshots

![Screenshot_2024-09-17-18-33-11-034_com cleverraven cataclysmdda experimental](https://github.com/user-attachments/assets/b0b9e081-19e9-4691-ba50-939c98543902)

### Versions and configuration

There is no such thing in my android version 

### Additional context

How are you supposed to move wrecks in this game?

**Comment by Falconinglasses** [comment]
![Screenshot_2024-09-17-18-50-06-025_com.cleverraven.cataclysmdda.experimental.jpg](https://github.com/user-attachments/assets/26c32c9e-bb7a-49ef-a28a-1b1ad613a856)

However it seems that extra light vehicles can be towed, but my V8 is too weak to tow 20 kilo bike. I guess the only way to move anything that cant move by itself is by smashing it with sledgehammer and then moving individual pieces of scrap.

**Comment by CoroNaut** [comment]
Unfortunately on my system, I can't reproduce. The v12 i spawned in can tow a cube van no problem. I was even able to get up to 170 before hitting the side of a building and just dying. Perhaps this is android only issue?
![image](https://github.com/user-attachments/assets/5b61c2e2-a1aa-49a3-9914-fc3da448932f)


[TESTINGONE-trimmed.tar.gz](https://github.com/user-attachments/files/17032408/TESTINGONE-trimmed.tar.gz)
- OS: Windows
    - OS Version: 10.0.19045.4894 (22H2)
- Game Version: cdda-experimental-2024-09-16-0441 e186f1b [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by RenechCDDA** [comment]
@Falconinglasses This absolutely needs a save attached.

**Comment by Falconinglasses** [comment]
@CoroNaut @RenechCDDA finally i did it, i pulled a save outta here(sorry it took so long, miui isnt very friendly to me and closest pc i can use without breaking a law and breaking into houses is about 70km away))

**Comment by PatrikLundell** [comment]
Note that towing only works if the vehicles have enough wheels. You can only tow a fairly limited piece of a wreck if  it doesn't have enough wheels.

**Comment by CoroNaut** [comment]
I downloaded your save and tryed dragging some wreckages around, if they didn't have enough wheels (like Patrik says), they didn't move. The superbike i hooked up to a v12 and for some reason it went north rather than the direction of i was driving, south. it didn't snap the cable... 

I also kind of expect that I should be able to tow even a 42kg electric scooter with only 1 wheel with a v12. But I can't. Any other vehicle with enough wheels got towed around just fine. I even collided one of the vehicles I was towing into a full wreck, and the wreck moved quite a distance from the impacts before it stopped getting hit by my towed vehicle. Definitely some jank here.

It would be reasonable to cut the power of your engine by even -75% if the towed vehicle doesn't have wheels, not being able to pull them at all is just weird. Bonus points if it makes a dirt trail when you pull vehicles across grass or flowers.

Towing an ice cream cart with its wheels taken off in a test world (1). it went north when I went west... and only snapped when I was about 30 ish tiles away (2). I then put the wheels back on and it followed me perfectly, but about 10 tiles away rather than 8 (3, 4). Anyways, very strange behavior, and not what I would expect.

[TESTINGONE-trimmed.tar.gz](https://github.com/user-attachments/files/17044762/TESTINGONE-trimmed.tar.gz)

1.
![image](https://github.com/user-attachments/assets/8bf4f547-1fb9-417e-a898-cda12c4ffc8e)
2.
![Untitled](https://github.com/user-attachments/assets/0aacf8b2-da19-487d-805f-7e1e5b616054)
3.
![image](https://github.com/user-attachments/assets/48673441-5c70-4b09-a68e-8567c270fd01)
4.
![image](https://github.com/user-attachments/assets/f626a8b6-7d84-49df-99b3-b94bd1a611c0)

- OS: Windows
    - OS Version: 10.0.19045.4894 (22H2)
- Game Version: cdda-experimental-2024-09-18-0513 7b773c1 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by Falconinglasses** [comment]
Yeah, by the cooment above is about what is going of with towing in this game, i could add tho, if towing wheeless wreks were to be a thing there could be even a mechanic to damage them further, even sometimes like
"Towing cable broke *steel frame*/*quarterpanel because of overload"
Because yes, i can tow a bicycle across the road, but the moment its gets to dirt(perfectly traversable(i van even Grab it and pull it with me)) it acts as a 10000000000 ton ultimate anvil becomes unmovable.

**Comment by Falconinglasses** [comment]
Yeah, by the cooment above is about what is going of with towing in this game, i could add tho, if towing wheeless wreks were to be a thing there could be even a mechanic to damage them further, even sometimes like
"Towing cable broke *steel frame*/*quarterpanel because of overload"
Because yes, i can tow a bicycle across the road, but the moment its gets to dirt(perfectly traversable(i van even Grab it and pull it with me)) it acts as a 10000000000 ton ultimate anvil becomes unmovable.
